src/components/Navbar/index.jsx

```diff
@@ -1,5 +1,6 @@
 "use client"
```

Comment:
same comment I added to the other PR apply here in regards to the component's internal organization and how you are using the `Switch`